app/controllers/podcasts_controller.rb

```diff
@@ -4,14 +4,16 @@ class PodcastsController < ApplicationController
   # Translate the user selected sort to a query order argument
   DISPLAY_ORDER = {"A-Z" => {title: :asc},
                    "Z-A" => {title: :desc},
-                   "" => {updated_at: :desc}}.freeze
+                   "" => {updated_at: :desc},
+                   "Recent Activity" => {updated_at: :desc}}.freeze
 
   DEFAULT_PAGE_SIZE = 10
 
   # GET /podcasts
   def index
     base_query = policy_scope(Podcast).page(params[:page]).per(DEFAULT_PAGE_SIZE).includes(default_feed: :feed_images)
-    @podcasts = add_sorting(base_query).filter_by_title(params[:q])
+    filtered_podcasts = base_query.filter_by_title(params[:q])
+    @podcasts = add_sorting(filtered_podcasts)
```

Comment:
Noting this change is functionally equivalent to what was there before.  (Calling both `add_sorting` and `.filter_by_title` on the base query, assigning the result to `@podcasts`).  But this is fine, if you think it's more readable this way.

Reply:
The latter way would maybe be easier for me to come back and understand what's happening, but I'm not insistent about it. I can change it back to the way it was before.